src/custom/pages/Claim/ClaimingStatus.tsx

```diff
@@ -0,0 +1,69 @@
+import { useMemo } from 'react'
+import { Trans } from '@lingui/macro'
+import { ConfirmOrLoadingWrapper, ConfirmedIcon, AttemptFooter } from 'pages/Claim/styled'
+import { ExternalLink, CustomLightSpinner } from 'theme'
+import { ClaimStatus } from 'state/claim/actions'
+import { useClaimState } from 'state/claim/hooks'
+import { useActiveWeb3React } from 'hooks/web3'
+import CowProtocolLogo from 'components/CowProtocolLogo'
+import Circle from 'assets/images/blue-loader.svg'
+import { formatSmart } from 'utils/format'
+
+export default function ClaimingStatus() {
+  const { chainId } = useActiveWeb3React()
+  const { activeClaimAccount, claimStatus, claimedAmount } = useClaimState()
+
+  // claim status
+  const isConfirmed = useMemo(() => claimStatus === ClaimStatus.CONFIRMED, [claimStatus])
+  const isAttempting = useMemo(() => claimStatus === ClaimStatus.ATTEMPTING, [claimStatus])
+  const isSubmitted = useMemo(() => claimStatus === ClaimStatus.SUBMITTED, [claimStatus])
```

Comment:
I feel like this useMemo adds more overhead that what it saves.
Not really an intensive calculation here, it's fine to do the comparison at every render IMO.

Reply:
Done